sql-schema/127.sql

```diff
@@ -0,0 +1,11 @@
+DROP TABLE IF EXISTS `ipmi_sensors`;
```

Comment:
I used the db to store information so that the script files won't have long lists of arrays with values.
Is a separate table so that it won't break or render obsolete existing informations (it is used for display only, no changes are made to discovery/polling).

in my case from HP ipmi i get only Temp 1, temp 2, temp 3, .... values but no names so they are pretty useless without knowing which is which.